app/decorators/models/solidus_afterpay/spree/order_decorator.rb

```diff
@@ -9,6 +9,10 @@ def available_payment_methods
         end
       end
 
+      def line_items_tax_amount
```

Comment:
For changes like this, that will be used only in the context of the gem, maybe it's better to use a presenter instead of patching the core. WDYT?

Reply:
Yes, I agree!